packages/now-cli/src/util/dev/server.ts

```diff
@@ -916,7 +916,6 @@ export default class DevServer {
     }
 
     ops.push(close(this.server));
-    ops.push(close(this.proxy));
```

Comment:
Let's update the close() method to look for `server.listening` and resolve immediately